### ISSUE TITLE
Doesn't Consider Fee

### ISSUE THREAD

**Issue by aadon94** [tony-ho/binance-oco#2]
Hi, first off nice project - thanks. Hopefully I can help contribute to it though I'm only a beginner programmer.

There's a sell error thrown after setting a limit and stop loss order.

```
BNBBTC BUY LIMIT ORDER #86467834 (FILLED)
..price: 0.00149770, quantity: 1.00000000
Sell error {"code":-2010,"msg":"Account has insufficient balance for requested action."}
```
Reason is Binance takes a fee and then we don't have enough to set that sell order.

**Comment by tony-ho** [comment]
@aadon94 Thanks for the feedback and for raising this issue.

The current behaviour is to reduce the sell amount by the fee paid, but only if the fee is not paid using BNB. This is to allow user who have selected to use BNB to pay fees (https://support.binance.com/hc/en-us/articles/115000583311) to trade the entire amount.

The logic is here: https://github.com/tony-ho/binance-oco/blob/3321bf41fa5b16e24104240cfb64b897631f316e/binance-oco.js#L161-L166

However, this causes an issue when:
- BNB is the asset being purchased eg. when trading BNBBTC, BNBETH, BNBUSDT pairs, and
- there is insufficient BNB in the account to cover fees before the buy order is filled, meaning the fee is deducted from the buy amount

The reason is that orders for BNB pairs will always pay fees using BNB. As the script does not reduce the sell amount when fees are paid using BNB, there will be insufficient balance for the sell order if the account does not already have enough BNB to pays fees.

I suspect this the cause of your error. Can you confirm if you had BNB balance in your account at the time of the error?

**Comment by tony-ho** [comment]
@aadon94 This issue has been addressed in v1.5.0.

This release adds a `-F`/`--non-bnb-fees` option to force non-BNB fee calculation ie. the sell order amount will be reduced by 0.1%.

**Comment by aadon94** [comment]
> I suspect this the cause of your error. Can you confirm if you had BNB balance in your account at the time of the error?
 
Yep that was the cause of it. Brilliant, thanks.
